src/generic/treebkg.cpp

```diff
@@ -564,6 +564,12 @@ wxWindow *wxTreebook::TryGetNonNullPage(size_t n)
     return page;
 }
 
+void wxTreebook::SetImages(const Images& images)
```

Comment:
We need to do here the same thing as in `wxListbook::OnImagesChanged()`, except we don't need to care about `wxLC_ICON` style, as we always use the icons here if they're specified.